draft-scenarios/clash/dragnarok.tex

```diff
@@ -50,7 +50,7 @@ \subsection*{\MakeUppercase{Defeat Conditions}}
 
 \subsection*{\MakeUppercase{Victory Points}}
 The player with the most Victory Points (VP) wins.
-In case of a tie, the player controlling more Dragon Utopias wins the game.
+In case of a tie, the tied players draw Cards to their \svg{hand} Limit and engage in a final Combat. The winner of that Combat wins the game.
```

Comment:
Maybe this explanation should be below the VP points? Firstly, you explain how players score VPs, and then the tiebreaker.